src/oss/confluent.sh

```diff
@@ -967,10 +979,10 @@ start_or_stop_service() {
     local entry=""
     for entry in "${list[@]}"; do
         "${command}"_"${entry}" "${@}";
-        [[ "${entry}" == "${service}" ]] && return 0
+        [[ "${entry}" == "${service}" ]] && return 1
     done
 
-    return 1
+    return 0
```

Comment:
would be good to document what these 1 and 0 here mean. It was not immediately obvious to me until you explained offline.